### ISSUE TITLE
Unable to get D: drive working on Unraid

### ISSUE THREAD

**Issue by rbisker** [JonathanTreffler/backblaze-personal-wine-container#66]
I'm running the docker on Unraid and after install executed the command
`docker exec --user app Backblaze_Personal_Backup ln -s /mnt/user /config
/wine/dosdevices/d:`

From the unraid shell I see a syslink "d:" in /mnt/user/appdata/Backblaze_Personal_Backup/wine/dosdevices, and when I browse with the ls command I see all of my shares listed.

However, when I start the docker, D: is not available as option in Backblaze.  Also, when I run `sudo docker exec --user app backblaze-backup-backblaze-1 wine explorer`, I see a D: drive listed, but it is empty.

![image](https://github.com/JonathanTreffler/backblaze-personal-wine-container/assets/31449097/c2717eb4-5159-4e48-a7a8-bc36ee2b4720)

Any idea what I'm doing wrong here?  Does this not work if you're making a symbolic link to Unraid share folders as opposed to directories on specific disks?

**Comment by henryyeh** [comment]
Permissions. Make sure the directory /mnt/user and symlink /wine/dosdevices/d: are owned by app:app, not root.

**Comment by rbisker** [comment]
Sorry I'm a Linux/Docker novice.  When you say change the user:group to app:app, do I literally type 'chown app:app' or is 'app' supposed to be the group ID of something in the docker, and if so how do I figure out what that is?

> Permissions. Make sure the directory /mnt/user and symlink /wine/dosdevices/d: are owned by app:app, not root.

**Comment by henryyeh** [comment]
Yes. Exec into the container with docker exec -it and literally chows them
with app:app. The wine process is owned and run by the “app” user.
